documentation/modules/deploying/proc-deploying-kafkaconnector.adoc

```diff
@@ -18,6 +18,11 @@ The file is used to create the following connector instances:
 * A `FileStreamSourceConnector` instance that reads each line from the Kafka license file (the source) and writes the data as messages to a single Kafka topic.
 * A `FileStreamSinkConnector` instance that reads messages from the Kafka topic and writes the messages to a temporary file (the sink).
 
+Until Apache Kafka 3.1.0, these connector plugins were always included in the Apache Kafka.
+Since Apache Kafka 3.1.1 and 3.2.0, they need to be added to the plugin path as any other connector.
```

Comment:
I think the specific versions are important here because of 3.1.1 => Without that, people will expect that its only 3.2.0 from which this changes because that is the logical step.

Reply:
Adjusted the suggestion